File: Python/lista2 - estruturas de repeticao/exercicio_6.py
count_numbers_between_30_and_90 = 0

for i in range(10):
    number = float(input('Informe um número: '))

    if number >= 30 and number <= 90:
        count_numbers_between_30_and_90 += 1

print(f'Quantidade de números entre 30 e 90: {count_numbers_between_30_and_90}')

File: C/lista2 - estrutura de repeticao/exercicio_6.c
#include <conio.h>
#include <stdio.h>

int main() {
	int count, i;
	float number;
	
	for (i= 0; i< 10; i++) {
		printf("Informe um numero: ");
		scanf("%f", &number);
		
		if (number >= 30 && number <= 90) {
			count++;
		}
	}
	
	printf("Quantidade de valores entre 30 e 90: ", count);
	
	getch();
	return 0;
}